lib/omnibus/packagers/bff.rb

```diff
@@ -229,6 +232,9 @@ def create_bff_file
       FileSyncer.glob(File.join(staging_dir, 'tmp/*.bff')).each do |bff|
         copy_file(bff, File.join(Config.package_dir, create_bff_file_name))
       end
+
+      # chown back to original user's uid/gid so cleanup works correctly
+      shellout!("sudo chown -Rh #{original_uid}:#{original_gid} #{staging_dir}")
```

Comment:
I'm wondering if this should be in an ensure block so we actually ensure ownership is changed back...even in the event  `mkinstallp` causes an exception to bubble up. 

```ruby
def create_bff_file
  ... # do original stuff here
ensure
  # chown back to original user's uid/gid so cleanup works correctly
  original_uid = shellout!("id -u").stdout.chomp
  original_gid = shellout!("id -g").stdout.chomp
  shellout!("sudo chown -Rh #{original_uid}:#{original_gid} #{staging_dir}")
end
```